server/game_service.py

```diff
@@ -154,15 +151,6 @@ def create_game(
             "matchmaker_queue_id": matchmaker_queue_id,
         }
         game_args.update(kwargs)
-
```

Comment:
I would definitely either make `game_class` a required argument, or give it a sensible default. Defaulting to `None` just doesn't make any sense anymore.